locales/en/messages.json

```diff
@@ -2957,9 +2957,13 @@
     "gpsAltitude": {
         "message": "Altitude:"
     },
-    "gpsLatLon": {
-        "message": "Current Latitude / Longitude:",
-        "description": "Show GPS position - Latitude / Longitude"
+    "gpsLatitude": {
+        "message": "Latitude:",
+        "description": "Show GPS position - Latitude"
+    },
+    "gpsLongitude": {
+        "message": "Longitude:",
+        "description": "Show GPS position - Longitude"
```

Comment:
The `Current` is not needed anymore? I think remember it was "included" some time ago to differentiate with the `Home Latitude / Longitude`.

Reply:
It seems it was a not merged PR... https://github.com/betaflight/betaflight-configurator/pull/3486